app/helpers/template_helpers.py

```diff
@@ -1,304 +1,379 @@
+
+        if (
+            self.context_options.powered_by_logo
+            or self.context_options.powered_by_logo_alt
+        ):
+            context["poweredBy"] = {
+                "logo": self.context_options.powered_by_logo,
+                "alt": self.context_options.powered_by_logo_alt,
+            }
+
+        return context
+
+    @cached_property
+    def _footer_warning(self) -> Optional[str]:
+        if self._is_post_submission:
+            return lazy_gettext(
+                "Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
+            ).format(sign_out_url=self._sign_out_url)
+
+
+@dataclass
+class CensusContextOptions(
+    ContextOptions,
+):
+    title_logo: str = "census-logo-en"
+    title_logo_alt: LazyString = lazy_gettext("Census 2021")
+    base_url: str = CENSUS_EN_BASE_URL
+    account_service_url = f"{base_url}/en/start"
+    design_system_theme = "census"
+    footer_links: Iterable[MutableMapping] = field(
+        default_factory=lambda: [
+            Link(
+                lazy_gettext("Help"),
+                f"{CENSUS_EN_BASE_URL}/help/help-with-the-questions/online-questions-help/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Contact us"), f"{CENSUS_EN_BASE_URL}/contact-us/"
+            ).__dict__,
+            Link(
+                lazy_gettext("Languages"),
+                f"{CENSUS_EN_BASE_URL}/help/languages-and-accessibility/languages/",
+            ).__dict__,
+            Link(
+                lazy_gettext("BSL and audio videos"),
+                f"{CENSUS_EN_BASE_URL}/help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/",
+            ).__dict__,
+        ],
+        compare=False,
+    )
+    footer_legal_links: Iterable[Mapping] = field(
+        default_factory=lambda: [
+            Link(lazy_gettext("Cookies"), f"{CENSUS_EN_BASE_URL}/cookies/").__dict__,
+            Link(
+                lazy_gettext("Accessibility statement"),
+                f"{CENSUS_EN_BASE_URL}/accessibility-statement/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Privacy and data protection"),
+                f"{CENSUS_EN_BASE_URL}/privacy-and-data-protection/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Terms and conditions"),
+                f"{CENSUS_EN_BASE_URL}/terms-and-conditions/",
+            ).__dict__,
+        ],
+        compare=False,
+    )
+    data_layer: Iterable[Mapping] = field(
+        default_factory=lambda: [{"nisra": False}], compare=False
+    )
 
-def _map_theme(theme: str) -> str:
-    """Maps a survey schema theme to a design system theme
 
-    :param theme: A schema defined theme
-    :returns: A design system theme
-    """
-    if theme and theme not in ["census", "census-nisra"]:
-        return "main"
-    return "census"
+@dataclass
+class CensusContextOptionsCymraeg(
+    CensusContextOptions,
```

Comment:
Does this use any properties defined in `CensusContextOptionsCymraeg`? Even if it does it might be clearer to decide that everything should inherit directly from `ContextOptions`.

Also, the class name should end with `ContextOptions`, so should be `CensusCymraegContextOptions` or `CymraegCensusContextOptions`.

Reply:
It uses title_logo_alt, design_system_theme, and survey_title